modules/c++/math.linear/unittests/test_mx.cpp

```diff
@@ -560,6 +604,8 @@ int main()
     TEST_CHECK(testPtrAssign);
     TEST_CHECK(testSTLVectorAssign);
     TEST_CHECK(testOrthoTranspose5x5);
+    TEST_CHECK(testNegateMxN);
+    TEST_CHECK(testNegate);
 
        
```

Comment:
Can you add a `return 0` at the end of `main()`?  Looks like we forgot to do this in the past.  Also can get rid of the extra newlines that are in this function for whatever reason if you want.